org.lflang/src/org/lflang/generator/c/CActionGenerator.java

```diff
@@ -0,0 +1,101 @@
+package org.lflang.generator.c;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import org.lflang.ASTUtils;
+import org.lflang.federated.FederateInstance;
+import org.lflang.generator.ActionInstance;
+import org.lflang.generator.CodeBuilder;
+import org.lflang.generator.GeneratorBase;
+import org.lflang.generator.ReactorInstance;
+import org.lflang.lf.Action;
+import org.lflang.lf.Reactor;
+import org.lflang.lf.ReactorDecl;
+
+
+public class CActionGenerator {
```

Comment:
This should have a class comment with a short description of the class and author information.